src/Symfony/Component/Yaml/Parser.php

```diff
@@ -217,7 +218,7 @@ private function doParse(string $value, int $flags)
                 }
 
                 if (!\is_string($key) && !\is_int($key)) {
-                    throw new ParseException(sprintf('%s keys are not supported. Quote your evaluable mapping keys instead.', is_numeric($key) ? 'Numeric' : 'Non-string'), $this->getRealCurrentLineNb() + 1, $this->currentLine);
+                    throw new ParseException(sprintf('"%s" keys are not supported. Quote your evaluable mapping keys instead.', is_numeric($key) ? 'Numeric' : 'Non-string'), $this->getRealCurrentLineNb() + 1, $this->currentLine);
```

Comment:
To be reverted (same for other changes suggested by fabbot, they are false positives)